_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -68566,8 +68614,8 @@
 /turf/open/floor/iron/grimy,
 /area/station/service/chapel/office)
 "wgx" = (
-/obj/item/kirbyplants/random,
 /obj/machinery/light/small/directional/south,
```

Comment:
Probably don't need three lights here:
![image](https://user-images.githubusercontent.com/63861499/170845113-9931578a-42f3-4b70-9be1-2a01ffd0e558.png)
I would keep the two on the right hand side.
```suggestion
```
Otherwise looks great!